index.html

```diff
@@ -305,8 +323,22 @@ <h4>App Web em 10 minutos com Web2py <span>Mauro Duarte</span></h4>
               </div>
             </div>
 
+
+            <div class="row schedule-item">
+              <div class="col-md-2"><time>20:45</time></div>
+              <div class="col-md-10">
+                <div class="speaker">
+                  <img src="img/speakers/larissa_yasin.jpg" alt="Larissa Yasin">
+                </div>
+                <h4>Realidade aumentada utilizando ARCore <span>Larisa Yasin</span></h4>
+                <p>
+                  O ARCore foi lançado pelo Google já faz um tempo e dá várias facilidades para o desenvolvimento de aplicações que utilizem realidade aumentada. Nessa talk veremos um pouco sobre como essa tecnologia funciona no Android e alguns exemplos de como aplicar realidade aumentada em apps.
```

Comment:
Please, break lines at 80th column, more or less.
Makes it ease to read on editors.